test/lit/passes/jspi.wast

```diff
@@ -0,0 +1,132 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: wasm-opt %s --jspi -all -S -o - | filecheck %s
+
+(module
+  ;; CHECK:      (type $externref_f64_=>_i32 (func (param externref f64) (result i32)))
+
+  ;; CHECK:      (type $f64_=>_i32 (func (param f64) (result i32)))
+
+  ;; CHECK:      (type $externref_i32_=>_i32 (func (param externref i32) (result i32)))
+
+  ;; CHECK:      (type $f64_=>_none (func (param f64)))
+
+  ;; CHECK:      (type $i32_=>_i32 (func (param i32) (result i32)))
+
+  ;; CHECK:      (import "js" "compute_delta" (func $import$compute_delta (param externref f64) (result i32)))
+  (import "js" "compute_delta" (func $compute_delta (param f64) (result i32)))
```

Comment:
Is this really the output from `update_lit_checks.py`? If so, I had forgotten that it handled imports and exports correctly.

Reply:
This is the output from `update_lit_checks.py`.